docs/modules/ROOT/pages/getting-started/configuration.adoc

```diff
@@ -0,0 +1,257 @@
+= Configuring KubeArchive
+
+This document explains how to configure KubeArchive so you can archive 
+resources and query them.
+
+== Prerequisites
+* KubeArchive installed and running in a cluster (xref:getting-started/installation.adoc[])
+* Know how to query the KubeArchive API (xref:getting-started/kubearchive-api.adoc[])
+
+== The KubeArchiveConfig resource
+
+To configure KubeArchive to archive or delete resources create a
+KubeArchiveConfig custom resource. KubeArchiveConfigs are limited to 1 per namespace
+and should be named `kubearchive`. KubeArchiveConfigs have this general form:
+
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+  namespace: default
+spec:
+  resources: [...]
+----
+
+`spec.resources` is a list of elements, each defining rules for a specific kind, so KubeArchive knows
+when to archive or delete them.
```

Comment:
This is a good place to use the [callouts feature](https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/#callout-syntax) from asciidocs.

```suggestion
spec:
  resources: [...] <1>
----
<1> `spec.resources` is a list of elements, each defining rules for a specific kind, so KubeArchive knows
when to archive or delete them.
```